gap/Bialgebroids.gi

```diff
@@ -463,6 +463,17 @@ InstallMethod( Algebroid,
     
 end );
 
+InstallMethod( BaseRing,
+        "for an algebroid",
+        [ IsAlgebroid ],
+
+  function( A )
+
+    return LeftActingDomain( UnderlyingQuiverAlgebra( B ) );
```

Comment:
You probably mean `UnderlyingQuiverAlgebra( B )`

Reply:
Yes. Thanks.